modules/odr_monitoring/Taskfile.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+
+tasks:
+  default:
```

Comment:
This is nice. Making a mental note to propogate to the other task files

Reply:
Yeah its a cool pattern I've adopted recently, I think it just gets rid of the error upon running `tasks`